hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/ResourceManager.java

```diff
@@ -1118,38 +1124,25 @@ protected void serviceStop() throws Exception {
     }
   }
 
-    /**
```

Comment:
The code needs to retain comments, why should we delete this part of the comments?

Reply:
Thanks for your reminder! It is necessary to address the Stopbug notices. I need to add a better design to track task execution results.  The checkstyle issues will also be addressed together in the follow-up.

Reply:
The SotopBug warning is to expect RMFatalToStandbyRunner to submit and wait for its execution result.
If waiting for the execution result synchronously results in a "cyclic dependency" issue; 
However, in the call method of TransitionToActiveStandbyRunnern, both execution results and exceptions have been uniformly processed and log printed. 
The RMFatalToStandbyRunner execution results only have two results: ① successful execution ② Execution exception failed, RM process exited.
My opinion is this warning can be ignored in this scene,  As adding a thread to wait for the result would be redundant.
@slfan1989  How does view and handle this warning? I would like to hear your opinion again.